src/lib/components/preview-table/config.ts

```diff
@@ -2,6 +2,7 @@ export const config = {
   defaultColumnWidth: 200,
   maxColumnWidth: 320,
   minColumnWidth: 120,
+  minHeaderWidth: 160,
```

Comment:
Does this functionally make the `minColumnWidth` 160? Because net `columnWidth` will equal `headerWidth`, right?

So, should this just be `minColumnWidth: 160`?

Reply:
I should clarify here, but it's a bit more complicated. Will circle back and change the names to reflect what's going on here